lib/api/v3/queries/schemas/query_filter_instance_schema_representer.rb

```diff
@@ -126,7 +124,7 @@ def dependencies
                                                                 operator,
                                                                 form_embedded: form_embedded)
 
-              hash[path] = value
+              hash[path] = value unless value.nil?
```

Comment:
This is the place, where the exclusion takes place, when the factory returns `nil`. It is not written to the hash, hence, it does not appear in the response.